templates/main.go

```diff
@@ -24,7 +24,7 @@ import (
 <%if eq (index .Params `billingEnabled`) "yes" %>	
 	"<% .Files.Repository %>/internal/billing"
 <%- end %>
-<%if neq (index .Params `cacheStore`) "none" %>	
+<%if eq (index .Params `cacheStore`) "none" %>
```

Comment:
This should be `ne`, so it can support both Memcached / redis, and it defaults to `none` 
same with the one below

Reply:
fixed